extension-requests/script.js

```diff
@@ -31,6 +37,34 @@ const render = async () => {
   await populateExtensionRequests({ status: Status.PENDING });
 };
 
+const initializeAccordions = () => {
+  let acc = document.getElementsByClassName('accordion');
+  let i;
+
+  for (i = 0; i < acc.length; i++) {
+    acc[i].addEventListener('click', function () {
+      this.classList.toggle('active');
+      let panel = this.nextElementSibling;
+      if (panel.style.maxHeight) {
+        panel.style.maxHeight = null;
+      } else {
+        closeAllAccordions();
+        panel.style.maxHeight = panel.scrollHeight + 'px';
+      }
+    });
+  }
+};
+const closeAllAccordions = () => {
```

Comment:
let panel = accordionsList[i].nextElementSibling;
This line of code is trying to get the next element sibling of the accordion element, but it is not working because the accordion element does not have a next element sibling. The correct line of code should be:

let panel = accordionsList[i].querySelector('.panel');

Reply:
<img width="717" alt="Screenshot 2023-08-11 at 1 49 34 AM" src="https://github.com/Real-Dev-Squad/website-dashboard/assets/98796547/7de86f49-9d3d-4c86-a146-a1abe611d03e">
This is the HTML code screenshot. Panel is the sibling the accordion button. Also please check the video attached where I've shown the working of accordion.